app/src/androidTest/java/org/mozilla/fenix/ui/SmokeTest.kt

```diff
@@ -422,10 +422,14 @@ class SmokeTest {
         }.goBackToHomeScreen {}
 
         navigationToolbar {
-        }.openTrackingProtectionTestPage(trackingPage.url, true) {}
+            // browsing a basic page to allow GV to load on a fresh run

```

Comment:
Nit: extra spaces here?

Reply:
It's the default format set by the project, otherwise lint would complain about it.